### ISSUE TITLE
Can't start meeting in Safari 12.1.2

### ISSUE THREAD

**Issue by ktarasov-clearscale** [aws/amazon-chime-sdk-js#629]
**Describe the bug**
I can start meeting in **Safari 13**, but it fails in **Safari 12.1.2**.

My scenario is
```ts
const audioVideo = session.audioVideo;

// I tried with null input audio device as commented below
// audioVideo.chooseAudioInputDevice(null);
// audioVideo.realtimeMuteLocalAudio();

audioVideo.start();

// get devices list (listAudioInputDevices / listVideoInputDevices / listAudioOutputDevices)
const devices = await this.getDevices();

audioVideo.chooseAudioInputDevice(devices[0] || null);
audioVideo.chooseVideoInputDevice(devices[1] || null);
audioVideo.chooseAudioOutputDevice(devices[2] || null);

audioVideo.stopLocalVideoTile();
audioVideo.removeLocalVideoTile();
audioVideo.realtimeMuteLocalAudio();
audioVideo.stopContentShare();
```

**Screenshots**
![Selection_791](https://user-images.githubusercontent.com/50212937/90376873-c80d3a80-e07f-11ea-917f-f0cf5a78a94c.png)

**Platform**
 - OS: MacOS Mojave
 - Browser: Safari
 - Browser Version: 12.1.2
 - Amazon Chime SDK GitHub Commit: v1.15.0

**Comment by ktarasov-clearscale** [comment]
Was resolved by starting **Safari ~12.1** session with selected audio device.
```ts
...
if (this._clientPlatform.is({ safari: "~12.1" })) {
  await audioVideo.chooseAudioInputDevice(this.audioInputList.first);
}
...
```

**Comment by RidipDe** [comment]
Great that it was fixed.
Closing the ticket for now.
